styles/newsletter.scss

```diff
@@ -113,12 +120,8 @@ $o-grid-gutter-small: oGridGutter(S);
 	margin: 14px 0 0;
 }
 
-.n-newsletter-signup--subscribed .n-newsletter-signup__submit {
-
-	@include oButtons($theme: primary);
-	@include oButtonsSize(big);
-	@include oButtonsBaseStyles;
-	margin: 14px 0 0;
+[data-newsletter-user-subscribed="true"] .n-newsletter-signup__submit {
+	@include oButtonsTheme(primary);
```

Comment:
why's `big` gone?

Reply:
Its been moved up into the base `.n-newsletter-signup__submit` as it was duplicated